src/main/drivers/system.c

```diff
@@ -31,6 +31,30 @@
 
 #include "system.h"
 
+const char *mcuTypeName =
+#if defined(STM32F1)
```

Comment:
I was thinking of adding support for reading the MCU type over MSP (to allow for simple extraction and reporting in analytics. Will probably do that as an `uint8_t` though rather than a string. What we could do is use the defines to assign the integer id, and then have a function returning the MCU type string based on this id. Thoughts?

Reply:
A variable holding the MCU type and a function returning corresponding string doesn't make sense if a firmware is built for a specific MCU... let me do what I can do.

Reply:
Good point, I was overthinking. Other options are to just use 2 defines / consts / functions, one for name, the other for numerical value to be returned over MSP, or to return the string over MSP.